### ISSUE TITLE
Write policy for starting/stopping/restarting rsyslog

### ISSUE THREAD

**Issue by ykhodorkovskiy** [TresysTechnology/clip#198]
```
type=USER_AVC msg=audit(1438547435.349:1921): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='avc:  denied  { start } for auid=1000 uid=0 gid=0 path="/usr/lib/systemd/system/rsyslog.service" cmdline="systemctl restart rsyslog.service" scontext=toor_u:toor_r:toor_t:s0 tcontext=system_u:object_r:journald_unit_file_t:s0 tclass=service  exe="/usr/lib/systemd/systemd" sauid=0 hostname=? addr=? terminal=?'
type=USER_AVC msg=audit(1438547435.659:1922): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='avc:  denied  { status } for auid=1000 uid=0 gid=0 path="/usr/lib/systemd/system/rsyslog.service" cmdline="systemctl restart rsyslog.service" scontext=toor_u:toor_r:toor_t:s0 tcontext=system_u:object_r:journald_unit_file_t:s0 tclass=service  exe="/usr/lib/systemd/systemd" sauid=0 hostname=? addr=? terminal=?'
```

**Comment by mpalmi** [comment]
Resolved with Issue #134 in mpalmi@5245a66d23f355b4f72eb3162df9546cb3a1b5cb

**Comment by ykhodorkovskiy** [comment]
Verified that this was resolved with #134. Leaving this open until #134 is merged.
